asset_manager/data/config.ini

```diff
@@ -1,5 +1,4 @@
 [DEFAULT]
-SHEET_ID = 1gv8CaVp7LsvaK4bXT566DDoAQ0bvadvRfBNvhhDHT9E
+SHEET_ID = 1G2ExVuly2b3XUPPDByijPYphUsLq-GsTiNGdHuLQ-UE
 SHEET_RANGE = Summary
 S3_BUCKET = ethanpswan-finances
-S3_SECRET_NAME = asset-manager-s3
```

Comment:
Unclear what this was for; I use the general s3 credentials on my local machine to access S3.

Reply:
Update – probably related to the `get_secret` function, which I also deleted